sql/src/main/java/io/crate/exceptions/SQLExceptions.java

```diff
@@ -176,6 +179,7 @@ private static Throwable esToCrateException(Throwable e) {
             return new UnsupportedFeatureException(e.getMessage(), (Exception) e);
         } else if (isDocumentAlreadyExistsException(e)) {
             return new DuplicateKeyException(
+                ((EngineException) e).getIndex().getName(),
```

Comment:
this is some kind of wrong api of ES, in practice this can never happen for a VersionConflictException